css/style.css

```diff
@@ -0,0 +1,54 @@
+body { 
+  display:  flex;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+body nav {
+  width: 70rem;
+}
+
+article {
+  margin-left: 19px;
+}
+header {
+  margin: 18px;
+}
+
+nav ul {
+  list-style: none;
+  padding: 0;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] Please increase the length of the ul element so the user can use the scrolling.


![image](https://user-images.githubusercontent.com/74744787/157310170-d9c6d555-7816-4571-b04a-a976aeb525a6.png)

Reply:
Kindly ask for changes in a more friendly and polite way instead of orders like that:
_Please Increase the length of the ul element so the user can use the scrolling._

Reply:
Kindly ask for changes in a more friendly and polite way instead of orders like that:
_Please Increase the length of the ul element so the user can use the scrolling._